vic/extensions/rout_irr/src/rout_alloc.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+#include <rout.h>
+#include <vic_def.h>
+#include <vic_run.h>
+#include <vic_driver_image.h>
+#include <netcdf.h>
+
+//define routing variables
+void rout_alloc(void){
+    extern rout_struct rout;
+    extern global_param_struct global_param;
+    extern domain_struct local_domain;
+    
+    rout.x_size=get_nc_dimension("/home/bram/VIC5.0Routing/VICTestSetup/input/VIC_Params/RVIC_input_NL.nc","lon");
+    rout.y_size=get_nc_dimension("/home/bram/VIC5.0Routing/VICTestSetup/input/VIC_Params/RVIC_input_NL.nc","lat");
+    rout.total_time=global_param.nrecs; 
+    rout.uh_length=96;
```

Comment:
Weet ik nog niet precies, nu staat dit nummer hier puur omdat hij ook al in de oude code werd gebruikt. Ik wil kijken of we misschien de uh per dag kunnen maken!